public_suffix_list.dat

```diff
@@ -11871,6 +11871,7 @@ usercontent.jp
 
 // Gentlent, Inc. : https://www.gentlent.com
 // Submitted by Tom Klein <tom@gentlent.com>
+cdn-edges.net
```

Comment:
@tomkln sorry to pedant this, please revise your change to organize the entries in your section ordered alphabetically by TLD then SLD ascending (.net after .com and .ms)

Reply:
Thanks for letting me know. I forgot that. **It's fixed** :)